apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/DatabaseTableFilter.java

```diff
@@ -1,19 +1,24 @@
 package com.linkedin.openhouse.jobs.util;
 
+import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
 import lombok.AllArgsConstructor;
 
 @AllArgsConstructor
 public class DatabaseTableFilter {
   private final Pattern databasePattern;
   private final Pattern tablePattern;
+  private final int minAgeThresholdHours;
 
-  public static DatabaseTableFilter of(String databaseRegex, String tableRegex) {
-    return new DatabaseTableFilter(Pattern.compile(databaseRegex), Pattern.compile(tableRegex));
+  public static DatabaseTableFilter of(
+      String databaseRegex, String tableRegex, int minAgeThresholdHours) {
```

Comment:
why do you need it if it's hard-coded to 0 afterwards?

Reply:
Thank you, it's a bug

Reply:
Lol https://github.com/linkedin/openhouse/pull/218/files#diff-5a1b364f28cc455bbb92dc5919303bd3046ff1bfd31cc6e42f02a4d2fb5e46c7R49

Reply:
Fixed